lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -72,9 +72,9 @@ def colorized_formatted_backtrace(colorizer=::RSpec::Core::Formatters::ConsoleCo
           end
         end
 
-        def fully_formatted(failure_number, colorizer=::RSpec::Core::Formatters::ConsoleCodes)
+        def fully_formatted(failure_number, colorizer=::RSpec::Core::Formatters::ConsoleCodes, skip=0)
           lines = fully_formatted_lines(failure_number, colorizer)
-          lines.join("\n") << "\n"
+          lines.drop(skip).join("\n") << "\n"
```

Comment:
That seems super implicit.  I'd prefer we go with something like `without_header: true` or `skip_example_description: true` over an implicit `skip` option that is based on knowledge of what the formatted form of the output is.

That said...I don't really want `fully_formatted` to gain a bunch of options to allow callers to customize how the formatting is done.  The point of `fully_formatted` is that it's the entire thing, and if you want to format it differently, the pieces are available to you to do that.

Instead, what if we didn't change the interface here at all, but instead got the fully formatted from in the HTML formatter, and then did something like:

``` ruby
without_header = fully_formatted.sub(/^#{example.full_description}\n\n/m, '')
```

(I assume that regex isn't quite right, but you get the point).  Or `fully_formatted.lines.drop_while { ... }` with some logic  or something.

That way, it's more clear what is being dropped than just "the first 2 lines, whatever they are".